common/src/main/java/com/example/demo/exception/BusinessException.java

```diff
@@ -0,0 +1,42 @@
+package com.example.demo.exception;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Getter
+@Setter
+public class BusinessException extends RuntimeException {
```

Comment:
Is there any reason you extend RuntimeException and not Exception?
By extending the first one, there's no requirement in declaring/catching the error which may lead  an another developer to not manage them.
By the way, it would be great to have an exception handler, handling this business exception as a Json output.